proto/edgir/elem.proto

```diff
@@ -96,6 +96,14 @@ message HierarchyBlock {
   bool is_abstract = 30;  // true if self_class is abstract, and should error if used in a design
 
   edgir.common.Metadata meta = 127;
+
+  message Phrases {
```

Comment:
Is there a more descriptive name we can give this? For example. StringInterpolatedDescription?